solr/core/src/java/org/apache/solr/cloud/autoscaling/AutoScalingHandler.java

```diff
@@ -542,7 +542,7 @@ private AutoScalingConfig handleSetTrigger(SolrQueryRequest req, SolrQueryRespon
     String eventTypeStr = op.getStr(EVENT);
 
     if (op.hasError()) return currentConfig;
-    TriggerEventType eventType = TriggerEventType.valueOf(eventTypeStr.trim().toUpperCase(Locale.ROOT));
+    TriggerEventType.valueOf(eventTypeStr.trim().toUpperCase(Locale.ROOT));
```

Comment:
I think you can remove the whole line.

Reply:
If the eventTypeStr is an invalid event type, it will throw an exception that there are no matches in TriggerEventType enum. I'll add a comment so the reason for this line is more obvious.